pkg/stream/reader.go

```diff
@@ -39,6 +39,7 @@ type KafkaConfig struct {
 	SaslMechanism   string `envconfig:"KAFKA_SASL_MECHANISM" default:"PLAIN"`
 	Topic           string `envconfig:"KAFKA_EVENT_STREAM_TOPIC" required:"true"`
 	GroupID         string `envconfig:"KAFKA_GROUP_ID" required:"true"`
+	DstTopic        string `envconfig:"KAFKA_EVENT_STREAM_TOPIC_DST" required:"true"`
```

Comment:
Does `Dst` stand for "Destination"?

Reply:
Yes it does, I will change to the full name